### PR TITLE
LPS-86313 Update ThingScreenlet with Apio Consumer 0.0.8 changes and small fixes

### PR DESCRIPTION
**Pull request by phcp**
@nhpatt @victorg1991 @dgarciasarai

### CODE REVIEW

File: android/samples/test-app/src/main/java/com/liferay/mobile/screens/testapp/postings/view/PersonDetailCustom.kt [hunk 1]
```diff
@@ -20,34 +20,34 @@ import android.text.method.LinkMovementMethod
...
-	override var thing: Thing? by converter<Person> {
-		avatar.thing = thing
-		name.text = it.name
-		email.text = Html.fromHtml("<a href=\"mailto:${it.email}\">${it.email}</a>")
-		email.linksClickable = true
-		email.movementMethod = LinkMovementMethod.getInstance()
-		jobTitle.text = it.jobTitle
-		birthDate.text = it.birthDate?.mediumFormat()
-	}
+    override var thing: Thing? by converter<Person> {
```
Comment (nhpatt) [inline android/samples/test-app/src/main/java/com/liferay/mobile/screens/testapp/postings/view/PersonDetailCustom.kt]: Are you sure we have the same detekt convention?

Comment (dgarciasarai) [reply android/samples/test-app/src/main/java/com/liferay/mobile/screens/testapp/postings/view/PersonDetailCustom.kt]: Yes, please! We can configure detekt for all the repositories we want and format all the code with the same convention 👍

Comment (phcp) [reply android/samples/test-app/src/main/java/com/liferay/mobile/screens/testapp/postings/view/PersonDetailCustom.kt]: Yes, I totally agree, it's a bit hard to rebase SF commits.

Comment (dgarciasarai) [reply android/samples/test-app/src/main/java/com/liferay/mobile/screens/testapp/postings/view/PersonDetailCustom.kt]: I'm preparing the format convention (the same as android consumer). We can review this pull after, ok?

Comment (phcp) [reply android/samples/test-app/src/main/java/com/liferay/mobile/screens/testapp/postings/view/PersonDetailCustom.kt]: Ok (:

**Comment by dgarciasarai** [comment]
I did the rebase, fixed the conflicts and ran detekt. Now is ready to merge :)

**Pull request merged**